### ISSUE TITLE
systemUsingDirectivesFirst false by default in Visual Studio Preview

### ISSUE THREAD

**Issue by LWehmschulteAtRosenxt** [DotNetAnalyzers/StyleCopAnalyzers#3896]
I've updated my Visual Studio 2022 to Preview Version 17.12.0 Preview 3.0 and since then the `systemUsingDirectivesFirst` configuration seems to behave as if the default value was false, which is different to what is documented in the [json schema](https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json)

The following code gives me the following Warning in Visual Studio:
`Using directives should be ordered alphabetically by the namespaces`
![image](https://github.com/user-attachments/assets/86dd3b13-cf12-4088-a372-1daec09fd7a9)

```cs
using System.Text.Json;
using ANameSpace;

var aClass = new AClass();

Console.WriteLine(JsonSerializer.Serialize(aClass));

namespace ANameSpace
{
    class AClass();
}
```

stylecop.json:
``` json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": { }
}
```

If I now manually set the systemUsingDirectivesFirst to true in the stylecop.json, the warning will disappear:
![image](https://github.com/user-attachments/assets/7c0cacdd-227b-491e-ac4d-e980e5992c65)

``` json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "orderingRules": {
      "systemUsingDirectivesFirst": true
    }
  }
}
```

**Comment by bjornhellander** [comment]
Thanks! I see it too.

If I don't  add the setting to style.json, I notice this as well:
* If I select Build Only in the Error List window after building, I actually see the expected behavior (based on which order the using directives are placed in).
* If I look at the Output window after building, I also see the expected behavior.

So it only seems to misbehave when running as part of the intellisense.

Do you see the same behaviour, @LWehmschulteAtRosenxt?

**Comment by bjornhellander** [comment]
I just upgraded to VS preview 5 and it seems to work as expected again. Could you close this if you agree?

**Comment by LWehmschulteAtRosenxt** [comment]
Yes. Same here. I'll close this. Thanks for your support

**Comment by sharwell** [comment]
This was fixed by https://github.com/dotnet/roslyn/pull/75317

**Comment by LWehmschulteAtRosenxt** [comment]
> This was fixed by [dotnet/roslyn#75317](https://github.com/dotnet/roslyn/pull/75317)

Thanks for your work 😊
